docs/getting-started/quickstart-programming.md

```diff
@@ -14,7 +14,7 @@ This tutorial assumes you've finished our [quickstart](./quickstart.md).
 For convenience, we recommend initializing a new DBOS application and starting a database for it:
 
 ```
-npx -y @dbos-inc/dbos-sdk@latest init -n <app-name>
+npm -y create @dbos-inc -- -n <app-name>
```

Comment:
I actually think `npx -y @dbos-inc/create -n <app-name>` is simpler--the double-dash is confusing.